pages/common/kafkacat.md

```diff
@@ -0,0 +1,35 @@
+# kafkacat
+
+> Apache Kafka producer and consumer tool.
+
+- Consume messages, start with the newest offset:
```

Comment:
starting. And remove the comma to match with the next description

Reply:
done